source/CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ add_subdirectory(physics)
 add_subdirectory(optimization/rol)
 add_subdirectory(non_matching)
 add_subdirectory(sundials)
+add_subdirectory(trilinos)
```

Comment:
Do we want to have the NOX wrappers in a stand-alone `trilinos` folder, or in `optimization/nox`? Maybe the latter, which places it next to Trilinos' `ROL` solver is more appropriate.

Reply:
Personally, I would prefer to keep it here. There are just too many options where place it and positions are very inconsistent in the library.